java/arcs/core/data/Annotation.kt

```diff
@@ -15,7 +15,7 @@ package arcs.core.data
  * An Arcs annotations containing additional information on an Arcs manifest element.
  * An Annotation may be attached to a plan, particle, handle, type etc.
  */
-data class Annotation(val name: String, val params: Map<String, AnnotationParam>) {
+data class Annotation(val name: String, val params: Map<String, AnnotationParam> = emptyMap()) {
 
     fun getParam(name: String): AnnotationParam {
```

Comment:
maybe this should be renamed as simply 'param' to match the pattern below where the getters like arcId/ttl/capability aren't prefixed?

Reply:
renamed